fine-mapping/SuSiE_RSS.ipynb

```diff
@@ -54,13 +54,13 @@
     "#The directory for output files\n",
     "parameter: cwd = path\n",
     "## The container with the lmm/marp software. Can be either a dockerhub image or a singularity `sif` file.\n",
-    "#parameter: container_lmm = '/home/tf437/scratch60/lmm_v_1_8.sif'\n",
+    "#parameter: container_lmm = '/home/hs863/LMM/lmm_v_1_3.sif'\n",
```

Comment:
please remove this line it is obslete. Also please use lmm:1.9 version of the docker image. it is the latest now.

Reply:
Should I replace it with 2.0 or remove the line altogether?

Reply:
let's just keep 2.0 and remove all the others